lab_3_keywords_textrank/main.py

```diff
@@ -152,8 +173,14 @@ def decode(self, encoded_tokens: tuple[int, ...]) -> Optional[tuple[str, ...]]:
                 sequence of string tokens
         In case of out-of-dictionary input data, None is returned
         """
-        pass
+        decoded_tokens = []
+        for token in encoded_tokens:
+            if token in self._id2word:
```

Comment:
revert the condition and you will not need to use `else`

Reply:
fixed